### PR TITLE
add maracasparrot

### PR DESCRIPTION
**Pull request by kevinhhuynh**

### CODE REVIEW

File: parrots.json [hunk 1]
```diff
@@ -684,4 +684,8 @@
       "gif": "braveheartparrot.gif",
       "name": "Braveheart Parrot"
     }
```
Comment (jmhobbs) [inline parrots.json]: Oops!  Needs a comma here 😄

Comment (kevinhhuynh) [reply parrots.json]: Oops, thanks!

**Comment by jmhobbs** [comment]
🎉

File: parrots.json [hunk 2]
```diff
@@ -683,5 +683,9 @@
     {
       "gif": "braveheartparrot.gif",
       "name": "Braveheart Parrot"
+    },
+    {
+      "gif": "hd/maracasparrot.gif",
```
Comment (jmhobbs) [inline parrots.json]: Whoops!  I missed this on the first pass, sorry.  It should be `"hd": "hd/maracasparrot.gif",` . Sorry!

Comment (kevinhhuynh) [reply parrots.json]: Ah I was wondering why it was failing all checks lol,

cheers

**Comment by jmhobbs** [comment]
Excellent, thank you!

**Pull request merged**